doc/lua_api.txt

```diff
@@ -2133,14 +2133,17 @@ Elements
 
 * Show an image
 
-### `animated_image[<X>,<Y>;<W>,<H>;<texture name>:<frame count>,<frame duration>]`
+### `animated_image[<X>,<Y>;<W>,<H>;<name>;<texture name>;<frame count>;<frame duration>;<frame start>]`
 
 * Show an animated image. The image is drawn like a "vertical_frames" tile
-    animation (See Tile animation definition), but uses a frame count/duration
+    animation (See [Tile animation definition]), but uses a frame count/duration
     for simplicity
-* `<texture name>` is the image to use
-* `<frame count>` is the number of frames animating the image
-* `<frame duration>` is in milliseconds
+* `<name>`: The event name. The event is the index of the current frame, and it is fired
```

Comment:
for consistency, don't include the `<>` when mentioning a formspec argument

Reply:
Sorry, did I miss anything? Nothing's changed for this line.